parts/linux/cloud-init/artifacts/components.json

```diff
@@ -44,7 +44,8 @@
       "multiArchVersions": [
         "v1.5.28",
```

Comment:
let's pull `v1.5.28`, `v1.5.32` should be fully rolled out by the time this VHD goes to AKS

Reply:
+ @AlisonB319 FYI

Reply:
I'm fine either way, seems like this PR addresses that now